### DISCUSSION TITLE
How To Add Dual Subtitle in MPV.NET

### DISCUSSION THREAD

**Discussion by metedeh** [mpvnet-player/mpv.net#714] [Q&A] [answered]
Hi guys, how to dual subtitle in mpv.net?
Subtitle 1 in top and subtile 2 in bottom

**Comment by stax76** [comment, marked as answer]
To get a GUI for the secondary sub, you can add this binding:

`Alt+b script-binding select/select-secondary-sid #custom-menu: Secondary Subtitle`

Alternatively, you can use the command palette user script:

https://github.com/stax76/mpv-scripts?tab=readme-ov-file#command_palette
